src/containers/WeightContainer/WeightContainer.jsx

```diff
@@ -152,10 +238,13 @@ class WeightContainer extends Component {
       setWeightLb(+value);
     } else {
       setWeight(+value);
+      setWeightLb(getWeightLb(+value));
     }
 
     this.setState({
       weightValue: value,
+      minWeight: units === 'cm' ? 30 : 66,
+      maxWeight: units === 'cm' ? 200 : 441,
     });
```

Comment:
make it variables as:

const MIN_WEIGHT_METRIC = 30;
const MIN_WEIGHT_IMPERIAL = 66;
const MAX_WEIGHT_METRIC....